### ISSUE TITLE
multiBarHorizontalChart overflowing container

### ISSUE THREAD

**Issue by WarrenR** [krispo/angular-nvd3#96]
When using a multiBarHorizontalChart without setting a range, as I don't know the range of the data that may be pushed to the chart, sometimes the bars overflow the container. 

It seems if I start with a high value, for example 60, it will adjust the axis on updates without issue up to a value like 100. But when it goes over it seems to stop adjusting the axis and just overflows the container.

It also happens if I start with a lower value, for example 0 or 5, it will adjust the axis until it reaches something like 11. 

So to recap when a range is set obviously if an update to the chart exceeds this range the bar will overflow. When a range is not set the chart will re-size based on the current max value in the data. However upon reaching certain values it fails to re-size the axis. The value it fails at seems to be in relation to the first update it gets that is higher than zero plus a certain amount. 

Hope I have made this clear.
